### PR TITLE
Issue 174 draft

### LINKED ISSUE #565: [174] Issue #174 - December 03, 2020
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/main/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/main/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by fassko**
Closes #565

### CODE REVIEW

File: _drafts/2020-12-03-issue-174.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+[Federico Zanetello](https://www.twitter.com/zntfdr) wrote a [blog post](https://fivestars.blog/swift/disfavoredOverload.html) about `@_disfavoredOverload`.
+
+> Function overloading is a powerful tool that enables us to define multiple functions of the same name with different implementations:
+in this article we’ve covered how Swift addresses ambiguity and how > we can (_cautiously_) use the `@_disfavoredOverload` attribute in case of multiple matches.
+
+[James Dempsey](https://twitter.com/jamesdempsey) announced a new project - [Swift Version](https://swiftversion.net/) - a website that shows which version of Swift shipped with which version of Xcode.
+
+[Mishal Shah](https://twitter.com/mishaldshah) announced [Swift Platform Support](https://swift.org/platform-support) - a central hub of detailing which tools and capabilities are available on specific platforms for the development and deployment of Swift applications.
```
Comment (morqon) [inline _drafts/2020-12-03-issue-174.md:42]: Ah, sorry, a typo: "a central hub of detailing" should be "a central hub detailing"

Thanks!

**Review by morqon — CHANGES_REQUESTED** [review]
Spotted a typo introduced in my PR. Suggested a correction.

File: _drafts/2020-12-03-issue-174.md (hunk 1 above)
Comment (jpsim) [inline _drafts/2020-12-03-issue-174.md:42]:
```suggestion
[Mishal Shah](https://twitter.com/mishaldshah) announced [Swift Platform Support](https://swift.org/platform-support) - a central hub detailing which tools and capabilities are available on specific platforms for the development and deployment of Swift applications.
```

Comment (jpsim) [reply _drafts/2020-12-03-issue-174.md]: Done.

**Comment by fassko** [comment]
Thank you @morqon and @jpsim for the corrections! 👍

**Pull request merged**